ProjectFiles/Assets/Scripts/Network/Events/ServerEliminatePlayersEvent.cs

```diff
@@ -5,12 +5,12 @@ namespace Network.Events {
     public class ServerEliminatePlayersEvent : Event {
         public ServerEliminatePlayersEvent() {
             ID = EventType.ServerEliminatePlayersEvent;
+            Length      = 2 * sizeof(ushort) + Players.Count * sizeof(int) + sizeof(byte);
```

Comment:
This can't be here will probably crash as Players will not have been initialised when deserialising the packet. needs to be after line 13 and 35

Reply:
Ah I see. I was wondering why we didn't put the length in the main constructor

Reply:
Done